### PR TITLE
fix: EDM4hep track covariance conversion

### PR DESCRIPTION
**Pull request by paulgessinger**
Blocked by:
- #2781

### CODE REVIEW

**Comment by paulgessinger** [comment]
This seems to be working again now

File: Tests/UnitTests/Plugins/EDM4hep/ConvertTrackEDM4hepTest.cpp [hunk 1]
```diff
@@ -365,7 +414,7 @@ BOOST_AUTO_TEST_CASE(RoundTripTests) {
                            readTs.smoothedCovariance(), 1e-5, 1e-6);
       Vector3 newCenter = readTs.referenceSurface().center(
           gctx);  // new center is a perigee, but should be on the other
-                  // surface
+      // surface
```
Comment (andiwand) [inline Tests/UnitTests/Plugins/EDM4hep/ConvertTrackEDM4hepTest.cpp:417]: I think the previous formatting was correct?

**Review by andiwand — APPROVED** [review]
cool :+1:

File: Tests/UnitTests/Plugins/EDM4hep/ConvertTrackEDM4hepTest.cpp (hunk 1 above)
Comment (asalzburger) [reply Tests/UnitTests/Plugins/EDM4hep/ConvertTrackEDM4hepTest.cpp:417]: I think that's ok like this as well.

**Comment by acts-project-service** [comment]
## :red_circle: Athena integration test results [456791a8a1aa1099c9d3543635a2f60e9682e16c]
### Build job with this PR failed!
Please investigate the [build job](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/35137103) for the [pipeline](https://gitlab.cern.ch/acts/acts-athena-ci/-/pipelines/6710170)!

**Pull request merged**